src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -38,7 +40,7 @@ export const JobLogsContainer = props => {
   );
 
   const customProperties = {
-    visibleColumns: DEFAULT_JOB_LOG_COLUMNS,
+    visibleColumns: setVisibleColumns(stripes),
```

Comment:
And use `stripes.hasPerm(perm)` to render visible columns, so you do not need to create additional function.
For example:
`visibleColumns: hasPermToDeleteLogs ? ['selected', ...DEFAULT_JOB_LOG_COLUMNS] : DEFAULT_JOB_LOG_COLUMNS`